tailwind.config.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
       black: "#1A1A1A",
       aqua: "#1890FF",
       grass: "#52C41A",
+      white: "#FFFFFF",
```

Comment:
white is already available within tailwind. and there is no need to add it here. I prefer only defining, what's missing, or if we are shifting the colors a bit like we did with `black`.